pkgs/top-level/all-packages.nix

```diff
@@ -20682,7 +20682,10 @@ with pkgs;
 
   wxmac = callPackage ../development/libraries/wxwidgets/wxmac30.nix { };
 
-  wxGTK31 = callPackage ../development/libraries/wxwidgets/wxGTK31.nix { };
+  wxGTK31 = callPackage ../development/libraries/wxwidgets/wxGTK31.nix {
+    withGtk2 = true;
```

Comment:
it should be withGTK2 = false,  the idea is to compile with GTK 3.

Reply:
The default in `wxGTK31.nix` is `withGtk2 ? true`. I changed it so that on darwin it's gtk3.